### PR TITLE
handwired/sono1: Teensy2.0++ revision, VIA keymap, documentation 

### PR DESCRIPTION
**Pull request by DmNosachev**
Another ASkeyboard Sono1 + Sono 2 numpad controller replacement project. I've used Teensy2.0++ this time as it almost perfectly fits to the footprint and pinout of original MCU.

## Description
1. Separated the revisions: `t2pp` (default) and `stm32f103`.
2. Made corresponding changes to `readme.md`.
3. Added `via` keymap.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/handwired/sono1/t2pp/rules.mk [hunk 1]
```diff
@@ -0,0 +1,2 @@
+MCU = at90usb1286
+BOOTLOADER = halfkay
```
Comment (fauxpark) [inline keyboards/handwired/sono1/t2pp/rules.mk:2]:
```suggestion
# MCU name
MCU = at90usb1286

# Bootloader selection
BOOTLOADER = halfkay
```

File: keyboards/handwired/sono1/stm32f103/mcuconf.h [hunk 2]
```diff
@@ -16,7 +16,7 @@
 
 /*
```
Comment (fauxpark) [inline keyboards/handwired/sono1/stm32f103/mcuconf.h:17]: These comments can probably go.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Comment by drashna** [comment]
Just a heads up, the stm32f103 version doesn't compile correctly for the via keymap, due to eeprom size.

**Pull request merged**